### PR TITLE
TE-7944 Added GA workflow, removed Travis.

### PR DESCRIPTION
**Pull request by stereomon**
Merge only

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+]
+
+jobs:
+  validation:
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version: [
+            '7.4',
```
Comment (dereuromark) [inline .github/workflows/ci.yml:15]: Best to add 7.3 here

Comment (stereomon) [reply .github/workflows/ci.yml:15]: 7.3 is covered in the prefer-lowest run

Comment (dereuromark) [reply .github/workflows/ci.yml:15]: Thats not enough though as that doesnt check the latest versions of dependencies against our code base in 7.3
So best to add that as separate run here.

**Comment by dereuromark** [comment]
The remaining issues must be false positives. I can check why CI reports them and fix those.
So this PR seems fine.

**Comment by dereuromark** [comment]
Since this is not a "module", CI doesn't work here.
So I removed it for now.

**Pull request merged**